src/backend/marsha/core/tests/test_api_video.py

```diff
@@ -3186,6 +3208,7 @@ def test_api_video_update_live_state_same_state(self):
             upload_state=PENDING,
             live_state=STOPPED,
             live_info={},
+            live_type=RAW,
```

Comment:
Not necessarily for this PR but I think that slowly adding all these fixed values is misusing factoryboy. The idea normally is to intelligently set random values for all fields in the factory and then, in your tests, set values only for a few fields depending on what you want to prove. I feel you are slowly adding fixed values to all your factory instanciations, defeating the purpose of random factories.

Reply:
Yes I totally forget to use `factoryboy`